### PR TITLE
Add adagio

### PR DESCRIPTION
**Pull request by rdmolony**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rdmolony** [comment]
Just waiting on triad to be installable on conda-forge, Ref conda-forge/staged-recipes#16044

**Comment by rdmolony** [comment]
@conda-forge-admin ready for review!  The linter doesn't seem to have registered that a newline has been re-added

File: recipes/adagio/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
```
Comment (dopplershift) [inline recipes/adagio/LICENSE:1]: This file is already included in the tarball on PyPI, so please remove from this PR.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Recipe is in great shape, just need to remove the unneeded license file (better to rely on the one that ships with the tarball). @kvnkho and @goodwanghan can you confirm you are willing to be listed as maintainers for this recipe?

File: recipes/adagio/LICENSE (hunk 1 above)
Comment (rdmolony) [reply recipes/adagio/LICENSE]: Thanks!

**Comment by rdmolony** [comment]
I didn't realise I could keep:

```yaml
  license: Apache-2.0
  license_file: LICENSE
```
without including the license file explicitely

**Comment by dopplershift** [comment]
Right. That's pointing to a file in the source directory. Since the tarball contains the file, it gets found and all is well. When you included it manually, you were essentially overwriting the copy that comes with the source.

**Comment by rdmolony** [comment]
I force pushed to resolve the new line linter issue - you seem to have removed the last new line in your edit which fixed it!  Weird

**Comment by dopplershift** [comment]
When you added that last line, it had trailing whitespace.

**Comment by kvnkho** [comment]
> Recipe is in great shape, just need to remove the unneeded license file (better to rely on the one that ships with the tarball). @kvnkho and @goodwanghan can you confirm you are willing to be listed as maintainers for this recipe?

Thanks for asking and yes I'm willing!

**Comment by goodwanghan** [comment]
> Recipe is in great shape, just need to remove the unneeded license file (better to rely on the one that ships with the tarball). @kvnkho and @goodwanghan can you confirm you are willing to be listed as maintainers for this recipe?

Thanks for asking and yes I'm willing!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**